tuned/plugins/plugin_scheduler.py

```diff
@@ -532,6 +545,7 @@ def _instance_init(self, instance):
 					instance._evlist.mmap(pages = perf_mmap_pages)
 			# no perf
 			except:
+				log.warn("python-perf unavailable, disabling perf support, you can try to (re)install python(3)-perf package")
```

Comment:
I believe `logging.warn` is deprecated in favor of `logging.warning`. We recently moved to using the latter. Also, it would be nice if the warning said that we are disabling runtime tuning.

Reply:
It seems I was used to it for very long :). Fixed and updated the string.